### DISCUSSION TITLE
threat score + known bots [OFF] = humans?

### DISCUSSION THREAD

**Discussion by mjau-mjau** [SocolSRT/cloudflare-rules#5] [Q&A]
Hi. Thanks for all the hard work on the Cloudflare rules. I am evaluating the rules briefly to have a basic understanding of what's going on, and I am curious about the below rule in **Threat Check - Challenge**:

```javascript
(cf.threat_score ge 10 and not cf.client.bot)
```

Isn't that like presenting a challenge to all human visitors with threat score > 10 (which is not that high)? Seems a bit overkill to challenge all human visitors regardless of any other condition for such a low threat score.

**Comment by SocolSRT** [comment]
Hello, maybe but I tested these rules on different sites and systems (wordpress, html, whmcs...), this parameter did not create any problems. You can of course change it to suit your site, as these rules are just an example, each site is unique

**Reply by mjau-mjau** [reply]
I was just wondering if this was the intended behavior of this rule. Thanks.
